main.go

```diff
@@ -2,15 +2,54 @@ package main
 
 import (
 	"fmt"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"log"
 	"net/http"
+	"os"
+	"time"
+)
+
+const (
+	evnVarPort = "PORT"
+	envVarAMQAddress = "AMQ_ADDRESS"
+	envVarAMQQueue = "AMQ_QUEUE"
 )
 
 func handler(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprintf(w, "Hello World!")
 }
```

Comment:
Do we still need this, or could we just remove it

Reply:
I'd say it can be removed